components/survey/Details/index.jsx

```diff
@@ -0,0 +1,70 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+
+import Icon from '@/components/Icon';
+import Button from '@/components/Button';
+
+import useUser from 'hooks/useUser';
+import useSurvey from 'hooks/useSurvey';
+import { getQuestionsFromSurvey } from './getQuestionsFromSurvey';
+
+export const surveyDetailsTestIds = {
+  base: 'survey-details',
+  coverImageUrl: 'survey-details__cover_image_url',
+  title: 'survey-details__title',
+  introText: 'survey-details__intro-text',
+  startSurveyButton: 'survey-details__start-survey',
+};
+
+const SurveyDetails = ({ setBackgroundImagePath }) => {
+  const router = useRouter();
+  const { id } = router.query;
```

Comment:
```suggestion
  const surveyId = router.query.id;
```

Would that work? This way we can explicitely say that it is the id of a survey (not of the user, or of the route). :)

Reply:
done in 0a12e6f2768d6cf6c93db4a407b806b7fc73a764